test/ci-test-weather.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Aren't the dependencies already defined in schema.json? If we cannot have it automatic, at least we can make users define just the list of dependencies. See https://github.com/smartsdk/dataModels/pull/14